coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -876,10 +881,11 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
    * Local helper to show the Delete Director confirmation popup.
    * @param director The director object to delete.
    */
-  private showDeleteDirectorConfirmation (director): void {
-    this.showPopup = true
-    this.activeDirectorToDelete = director
-  }
+  // removed until release 2
+  // private showDeleteDirectorConfirmation (director): void {
+  //   this.showPopup = true
+  //   this.activeDirectorToDelete = director
+  // }
```

Comment:
Just to clarify, what is the state of the commented out code in the Directors component. Is it something that is in the works in the near future?

Reply:
I'm not certain about future plans for the Directors component, although I've overhead "custom appointment/cessation dates" and some of the commented-out code refers to that as well.
I don't know what else to say here. I don't want to delete the work done by another developer even though it's not currently in use. I commented out the method above because it was called from commented-out template code, and I didn't want to pretend that it is currently valid code.

Reply:
Fair enough, thanks!